codalab/rest/bundle.py

```diff
@@ -61,6 +65,40 @@ def get_blob(uuid, path=''):
     return fileobj
 
 
+@put('/bundles/<uuid:re:%s>/contents/blob/' % spec_util.UUID_STR,
+     apply=AuthenticatedPlugin())
+def update_blob(uuid):
+    """
+    Update the contents of the given running or uploading bundle.
+
+    Accepts the filename as a query parameter, used to determine whether the
+    upload contains an archive.
+
+    Bundle state is updated to READY after upload is complete if the |finalize|
```

Comment:
Not implemented. When you do implement, change to FAILED if there is an exception during upload.